templates/helm/command-center/values.schema.json

```diff
@@ -13,5 +17,5 @@
       "required": ["name", "project", "region"]
     }
   },
-  "required": ["env", "cloudsql"]
+  "required": ["env", "cloudsql", "argoNamespaces"]
```

Comment:
Really not an issue at all but if you would want to, a nitpickiest of nitpicks might be to put "argoNamespaces" between "env" and "cloudsql" in the "required" list for the sake of ordering them the same lol. NOT IMPORTANT